### ISSUE TITLE
New, updated home page copy

### ISSUE THREAD

**Issue by m-i-k-e** [IntersectMBO/govtool#1903]
We need to change copy to better serve our audience, and to improve the description of what GovTool does.

**Comment by m-i-k-e** [comment]
This is the updated copy to replace the current homepage copy:

> Cardano Governance lets ADA holders have a say in the network's future. 

You can vote directly, find someone to represent you, or become a representative for others.

You can also propose changes in the network yourself, put these up for discussion and eventually, a vote.

[Learn more about governance](docs.gov.tools) in the guide, or have a look around the app and see Cardano Governance in action.

**Comment by m-i-k-e** [comment]
![Landing page - 1442](https://github.com/user-attachments/assets/e283dddb-85e8-4c75-bf7d-154a2e4ecf9f)

**Comment by Ryun1** [comment]
I dont think we should capitalise ADA, that is the ticker, the currency is Ada / ada
[see context](https://github.com/Ryun1/CIPs/tree/cip-constitution-tech/CIP-0120#capitalisation-of-key-terms)

**Comment by jdyczka** [comment]
@Ryun1 @m-i-k-e Maybe there sure be a separate issue for it, since there are other occurrences in the copy in other places as well:
- a few more `ADA holders`\`ADA Holders` -> `Ada holders`
-`... which is equal to the balance of ADA in their connected wallet -> `ada`
- `You do not currently have enough ADA in your wallet to continue. -> `ada`
- `You will receive a refund of {{deposit}} ADA when the transaction  completes` -> not sure it should be `ADA` or `ada`
- `Your deposit of {{deposit}} ADA will be refunded` -> as above
- `... your Voting Power will consist of your own ADA balance ...` -> `Ada balance`?

What do you think?

**Comment by m-i-k-e** [comment]
Yes @bosko-m

**Comment by Ryun1** [comment]
@jdyczka @m-i-k-e 
ill make a separate issue

**Comment by Ryun1** [comment]
New issue, lets take discussion there
https://github.com/IntersectMBO/govtool/issues/1916
